apps/spruce/src/pages/waterfall/DateFilter/index.tsx

```diff
@@ -38,6 +39,7 @@ export const DateFilter = () => {
       data-cy="date-picker"
       label="Go to Date"
       max={new Date()}
+      min={subDays(subYears(new Date(), 1), 1)}
```

Comment:
(regarding test failures) maybe we should remove this limitation for local & dev environments? Even if we update the test data, if a year passes, we will run into the same failures